### PR TITLE
Add GIF Brewery.app v3.0-b3

### PR DESCRIPTION
**Pull request by sebnash**

### CODE REVIEW

File: Casks/gif-brewery3-beta.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'gif-brewery3-beta' do
+  version '3.0-b3'
+  sha256 'b017f00a6a8f3a563ae74d405d2140d2a5ea1a438c482a4ce51716785d2df254'
+
+  url "http://www.helloresolven.com/betas/gifbrewery/GIFBrewery-#{version}.zip"
+  homepage 'http://www.helloresolven.com/betas/gifbrewery/'
+  license :gratis
+
+  app "GIFBrewery-#{version}/GIFBrewery-3.0-beta.app"
```
Comment (vitorgalvao) [inline Casks/gif-brewery3-beta.rb]: You can change the `3.0` to `#{version.sub(%r{-.*},'')}`.

**Comment by sebnash** [comment]
@vitorgalvao ah! Done.

**Comment by vitorgalvao** [comment]
Thank you. We hope to have some better ways to do these kinds of operations for the most common cases, in the future.

Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by sebnash** [comment]
@vitorgalvao sure. Done.

**Comment by vitorgalvao** [comment]
Thank you. Merged.

**Pull request merged**